DOCS/man/options.rst

```diff
@@ -3819,6 +3819,31 @@ Demuxer
     (This value tends to be fuzzy, because many file formats don't store linear
     timestamps.)
 
+``--demuxer-hysteresis-secs=<seconds>``
+    Once the ``--demuxer-max-bytes`` prefetch limit is reached, this value can
+    be used to specify a hysteresis before the demuxer will prefetch again. This
+    specifies the maximum number of seconds that needs to be remaining in the
+    cache before prefetching will resume.
+
+    For example, with a value of 10 seconds specified, the demuxer will prefetch
+    up to ``--demuxer-max-bytes`` and won't start prefetching again until there
+    is only 10 seconds of content left in the cache. When the demuxer starts
```

Comment:
might be worth clarifying that this doesn't take the cache behind the current read position into account